### PR TITLE
Marketing pages: misc styling + remake table for download page

### PR DESCRIPTION
**Pull request by tarafanlin**
making this PR so Will can get the updated version and fix the safari bug on top of it.

### CODE REVIEW

File: pages/_/marketing-candidate/download.js [hunk 1]
```diff
@@ -17,11 +16,13 @@ const STYLES_ROOT = css`
 
 const STYLES_H1 = css`
   font-size: ${Constants.typescale.lvl5};
+  font-weight: 700;
```
Comment (jimmylee) [inline pages/_/marketing-candidate/download.js:19]: Take font-weight out, again use the font-family.

File: pages/_/marketing-candidate/download.js [hunk 2]
```diff
@@ -48,17 +50,18 @@ const STYLES_H3 = css`
...
   justify-content: space-between;
   max-width: 56vw;
   width: 100%;
   align-self: center;
+  margin-bottom: 16px;
 
   @media (max-width: ${Constants.sizes.mobile}px) {
     width: 100%;
-    right: 24px;
+    max-width: 100vw;
```
Comment (jimmylee) [inline pages/_/marketing-candidate/download.js:64]: What does this do?

File: pages/index.js [hunk 3]
```diff
@@ -165,6 +138,7 @@ const STYLES_LINK = css`
 
 const STYLES_H1 = css`
   font-size: ${Constants.typescale.lvl5};
+  font-weight: 700;
```
Comment (jimmylee) [inline pages/index.js:141]: Don't use font-weight, use font-family and add the fonts you need.

File: pages/index.js [hunk 4]
```diff
@@ -183,6 +157,7 @@ const STYLES_H1 = css`
 
 const STYLES_H2 = css`
   font-size: ${Constants.typescale.lvl3};
+  font-weight: 500;
```
Comment (jimmylee) [inline pages/index.js:160]: Don't use font-weight unless you're using a system font that has the weightings, we use inter and use a specific woff per font-weight.

File: pages/index.js [hunk 5]
```diff
@@ -201,6 +176,7 @@ const STYLES_H2 = css`
 
 const STYLES_H3 = css`
   font-size: ${Constants.typescale.lvl2};
+  font-weight: 500;
```
Comment (jimmylee) [inline pages/index.js:179]: See above

**Review by jimmylee — CHANGES_REQUESTED** [review]

File: pages/index.js (hunk 3 above)
Comment (tarafanlin) [reply pages/index.js:141]: wait I thought you wanted us to add `font-weight: 400` yesterday? Is there a difference?

File: pages/_/marketing-candidate/download.js (hunk 2 above)
Comment (tarafanlin) [reply pages/_/marketing-candidate/download.js:64]: to overite the `max-width: 56vw` in desktop

**Review by jimmylee — APPROVED** [review]

**Pull request merged**